### PR TITLE
Modify UI

### LINKED ISSUE #134: Dark Mode UI

### PR DESCRIPTION
**Pull request by ph-nathan**
Closes #134

What have you done in this PR?
* Refactor css file to remove redundant styleClass
* Add dark mode and logic for switching theme
* Fix gray box for empty events in light mode

Dev Testing Steps / Screenshots
image

### CODE REVIEW

File: src/test/java/seedu/ccacommander/storage/JsonUserPrefsStorageTest.java
Comment (wr1159) [inline src/test/java/seedu/ccacommander/storage/JsonUserPrefsStorageTest.java:1]: Why did you remove these 2 test cases? I understand some parts of the UI cannot be tested and the absence of test is understandable but I don't get the removal of test cases

**Review by wr1159 — APPROVED** [review]
I think we should save the theme selection into the userpreference json file. Currently, I can select the light theme and once I close the app, it will reset as dark mode again. I think if the preference is not saved, this will be counted as a bug as it will cause frustrations among the users.

I added a comment on the JsonUserPrefsStorageTest which is relevant to above too.

Aside from that, I like the dark mode, good job!

File: src/test/java/seedu/ccacommander/storage/JsonUserPrefsStorageTest.java
Comment (ph-nathan) [reply src/test/java/seedu/ccacommander/storage/JsonUserPrefsStorageTest.java:1]: the removal was due to the testcases failing, because the themes wasn't stored in UserPref Json file. Now that you've mentioned that it is a possible bug if the theme isn't saved, I will modify and reinclude the testcases. Thanks for the feedback!

**Review by dylkaw — APPROVED** [review]
LGTM now that test cases have been added!
